wiki/content/deploy/dgraph-zero.md

```diff
@@ -62,121 +76,90 @@ The `/state` endpoint of Dgraph Zero returns a JSON document of the current grou
 - Max Leased UID.
 - CID (Cluster ID).
 
-Here’s an example of JSON returned from `/state` endpoint for a 6-node Dgraph cluster with three replicas:
+Here’s an example of JSON returned from the `/state` endpoint:
 
 ```json
 {
-  "counter": "15",
-  "groups": {
-    "1": {
-      "members": {
-        "1": {
-          "id": "1",
-          "groupId": 1,
-          "addr": "alpha1:7080",
-          "leader": true,
-          "lastUpdate": "1576112366"
-        },
-        "2": {
-          "id": "2",
-          "groupId": 1,
-          "addr": "alpha2:7080"
-        },
-        "3": {
-          "id": "3",
-          "groupId": 1,
-          "addr": "alpha3:7080"
-        }
-      },
-      "tablets": {
-        "counter.val": {
-          "groupId": 1,
-          "predicate": "counter.val"
-        },
-        "dgraph.type": {
-          "groupId": 1,
-          "predicate": "dgraph.type"
-        }
-      },
-      "checksum": "1021598189643258447"
-    }
-  },
   "zeros": {
     "1": {
       "id": "1",
-      "addr": "zero1:5080",
-      "leader": true
+      "groupId": 0,
+      "addr": "localhost:5080",
+      "leader": true,
+      "amDead": false,
+      "lastUpdate": "0",
+      "clusterInfoOnly": false,
+      "forceGroupId": false
     },
     "2": {
       "id": "2",
-      "addr": "zero2:5080"
-    },
-    "3": {
-      "id": "3",
-      "addr": "zero3:5080"
+      "groupId": 0,
+      "addr": "localhost:5081",
+      "leader": false,
+      "amDead": false,
+      "lastUpdate": "0",
+      "clusterInfoOnly": false,
+      "forceGroupId": false
     }
   },
-  "maxLeaseId": "10000",
+  "maxLeaseId": "0",
   "maxTxnTs": "10000",
-  "cid": "3602537a-ee49-43cb-9792-c766eea683dc",
+  "maxRaftId": "2",
+  "removed": [],
+  "cid": "cdcb1edb-8c81-4557-af99-ebed2b383e3c",
   "license": {
+    "user": "",
     "maxNodes": "18446744073709551615",
-    "expiryTs": "1578704367",
+    "expiryTs": "1597232699",
     "enabled": true
   }
 }
 ```
 
```

Comment:
The full json looks like this. I didn't post the full json in the JIRA ticket. Sorry, my bad.
```json
{
  "counter": "22",
  "groups": {
    "1": {
      "members": {
        "1": {
          "id": "1",
          "groupId": 1,
          "addr": "alpha2:7082",
          "leader": true,
          "amDead": false,
          "lastUpdate": "1603350485",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "2": {
          "id": "2",
          "groupId": 1,
          "addr": "alpha1:7080",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "3": {
          "id": "3",
          "groupId": 1,
          "addr": "alpha3:7083",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        }
      },
      "tablets": {
        "dgraph.cors": {
          "groupId": 1,
          "predicate": "dgraph.cors",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_created_at": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_created_at",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_history": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_history",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.xid": {
          "groupId": 1,
          "predicate": "dgraph.graphql.xid",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.type": {
          "groupId": 1,
          "predicate": "dgraph.type",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        }
      },
      "snapshotTs": "22",
      "checksum": "18099480229465877561"
    }
  },
  "zeros": {
    "1": {
      "id": "1",
      "groupId": 0,
      "addr": "zero1:5080",
      "leader": true,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "2": {
      "id": "2",
      "groupId": 0,
      "addr": "zero2:5082",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "3": {
      "id": "3",
      "groupId": 0,
      "addr": "zero3:5083",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    }
  },
  "maxLeaseId": "10000",
  "maxTxnTs": "10000",
  "maxRaftId": "3",
  "removed": [],
  "cid": "2571d268-b574-41fa-ae5e-a6f8da175d6d",
  "license": {
    "user": "",
    "maxNodes": "18446744073709551615",
    "expiryTs": "1605942487",
    "enabled": true
  }
}
```
This is for cluster with 3 alpha and 3 zeros. If this looks very big for the doc, we can figure out if we should show output for a smaller cluster or something else.